packages/msgpack_c.rb

```diff
@@ -1,50 +1,38 @@
 # Adapted from Arch Linux msgpack-c PKGBUILD at:
 # https://github.com/archlinux/svntogit-community/raw/packages/msgpack-c/trunk/PKGBUILD
 
-require 'package'
+require 'buildsystems/cmake'
 
-class Msgpack_c < Package
+class Msgpack_c < CMake
   description 'An efficient object serialization library'
   homepage 'https://msgpack.org/'
-  version '4.0.0'
+  version '6.0.0'
   license 'custom:Boost'
```

Comment:
From what I can see on the repo, its just the version 1.0 of the Boost Software License (BSL-1.0), nothing custom.